modules/internal/ArrayViewSlice.chpl

```diff
@@ -177,9 +177,10 @@ module ArrayViewSlice {
 
     iter these(param tag: iterKind) ref
       where tag == iterKind.standalone && !localeModelHasSublocales &&
-           __primitive("method call resolves", privDom, "these", tag) {
-      const ref myarr = arr;
-      forall i in privDom do yield myarr.dsiAccess(i);
+      __primitive("method call resolves", privDom, "these", tag) {
```

Comment:
I like the previous indentation better, to separate the where-clause from the function body.

Reply:
This PR isn't ready for review or merging.  It's a draft.  Stay on target red 5.